packages/modelviewer.dev/styles/docs.css

```diff
@@ -81,6 +81,7 @@ hr.solid {
 .tab {
   display: flex;
   align-items: center;
+  padding-right: 10px;
```

Comment:
The style is looking good. Only thing I noticed is when you click on a sidebar item on mobile it turns gray (and stays gray) like it's somehow getting the hover state applied.

Reply:
I think this is a consistent mobile feature. I could have the sidebar close after the click, which would avoid that.

Reply:
Yes, I think closing makes sense, especially because you can't really see the content has changed now.

Reply:
done in new pr